packages/hash/frontend/src/components/hooks/useAccountPages.ts

```diff
@@ -19,26 +19,28 @@ export type AccountPagesInfo = {
   loading: boolean;
 };
 
-export const useAccountPages = (accountId: string): AccountPagesInfo => {
+export const useAccountPages = (ownedById: string): AccountPagesInfo => {
   const { data, loading } = useQuery<
     GetAccountPagesTreeQuery,
     GetAccountPagesTreeQueryVariables
   >(getAccountPagesTree, {
-    variables: { accountId },
+    variables: { ownedById },
   });
 
   const accountPages = useMemo(() => {
     if (!data) {
       return [];
     }
 
-    return data?.accountPages.map(
-      ({ entityId, parentPageEntityId, properties: { title, index } }) => {
+    return data?.knowledgePages.map(
+      ({ entityId, parentPage, title, index }) => {
```

Comment:
parentPageModel?

Reply:
This is in the frontend and using the return type from the GQL API, where we haven't made any references to the word "Model" or the model classes.

Reply:
Oh so it is, sorry